rubygem/lib/zeus/rails.rb

```diff
@@ -182,6 +182,14 @@ def test_environment
     end
 
     def test_helper
+      # don't let minitest setup another exit hook
+      begin
+        require 'minitest/unit'
+        MiniTest::Unit.class_variable_set("@@installed_at_exit", true)
```

Comment:
Hi @zenspider I know this is a bit of a necro comment thread but I wanted to engage you a bit on this. You're 100% right this is a hack and it doesn't work right in Minitest 5. We introduced a patch for that as well, though in my experience there are still issues.

We are running Minitest tests through a modified version of [qrush's m gem](https://github.com/qrush/m) by passing ARGV through to it so that we can run tests by line number. As far as I know, Minitest does not support this.